sparkmagic/sparkmagic/livyclientlib/command.py

```diff
@@ -38,7 +40,11 @@ def execute(self, session):
         statement_id = -1
         try:
             session.wait_for_idle()
-            data = {u"code": self.code}
+            try:
+                codekind = conf.get_livy_kind(self.language)
+            except BadUserConfigurationException as e:
+                codekind = session.kind
```

Comment:
Why do we need to default this to the session kind? A statement without any kind attribute will use the session kind by default. Reference: https://livy.incubator.apache.org/docs/latest/rest-api.html#session-kind